circle.yml

```diff
@@ -0,0 +1,16 @@
+
+machine:
+  node:
+    version: 8
+
+test:
+  override:
+    - make verify
+    - make test
+
+deployment:
+  release:
+    tag: /v.*/
+    owner: Financial-Times
+    commands:
+      - make npm-publish
```

Comment:
This currently requires you to add additional commands into the Circle UI (which I'd like to switch all of our modules away from). A better example to follow is: https://github.com/Financial-Times/origami-service-makefile/blob/master/circle.yml